src/pages/blog/welcome-tsc-members-2025.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Welcoming New TSC Members"
+tags: ["foundation"]
+date: 2025-2-5
```

Comment:
I don't know how strict the date parser is... Better safe than sorry :man_shrugging: 

```suggestion
date: 2025-02-05
```